python3/koans/about_iteration.py

```diff
@@ -120,18 +122,11 @@ def test_all_iteration_methods_work_on_any_sequence_not_just_lists(self):
         result = map(self.add_ten, range(1,4))
         self.assertEqual(__, list(result))
 
-        try:
-            file = open("example_file.txt")
-
-            try:
-                def make_upcase(line):
-                    return line.strip().upper()
-                upcase_lines = map(make_upcase, file.readlines())
-                self.assertEqual(__, list(upcase_lines))
-            finally:
-                # Arg, this is ugly.
-                # We will figure out how to fix this later.
-                file.close()
-        except IOError:
-            # should never happen
-            self.fail()
```

Comment:
I think once upon time I was demonstrating all this try/catchery as a leadup to demonstrating another technique further in, but come back to this fresh, I'm inclined to agree with the sentiment of simpliying things.